client/extensions/woocommerce/app/orders/orders-list.js

```diff
@@ -0,0 +1,194 @@
+/**
+ * External dependencies
+ */
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import React, { Component } from 'react';
+import { times } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import EmptyContent from 'components/empty-content';
+import { fetchOrders } from 'woocommerce/state/sites/orders/actions';
+import {
+	areOrdersLoading,
+	areOrdersLoaded,
+	getOrders,
+	getTotalOrdersPages
+} from 'woocommerce/state/sites/orders/selectors';
+import { getOrdersCurrentPage } from 'woocommerce/state/ui/orders/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteAdminUrl } from 'state/sites/selectors';
+import { setCurrentPage } from 'woocommerce/state/ui/orders/actions';
+import Table from 'woocommerce/components/table';
+import TableRow from 'woocommerce/components/table/table-row';
+import TableItem from 'woocommerce/components/table/table-item';
+
+class Orders extends Component {
+	componentDidMount() {
+		const { siteId, currentPage } = this.props;
+
+		if ( siteId ) {
+			this.props.fetchOrders( siteId, currentPage );
+		}
+	}
+
+	componentWillReceiveProps( newProps ) {
+		if ( newProps.currentPage !== this.props.currentPage || newProps.siteId !== this.props.siteId ) {
+			this.props.fetchOrders( newProps.siteId, newProps.currentPage );
+		}
+	}
+
+	getOrderStatus = ( status ) => {
+		const { translate } = this.props;
+		const classes = `orders__item-status is-${ status }`;
+		let paymentLabel;
+		let shippingLabel;
+		switch ( status ) {
+			case 'pending':
+				shippingLabel = translate( 'New order' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Save New Order' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/woocommerce-order-list)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).